testing/acceptance_test.py

```diff
@@ -730,8 +730,9 @@ def test_cmdline_python_namespace_package(self, testdir, monkeypatch):
         assert result.ret == 0
         result.stdout.fnmatch_lines(
             [
-                "test_hello.py::test_hello*PASSED*",
-                "test_hello.py::test_other*PASSED*",
+                "rootdir: {}/world".format(testdir.tmpdir),
+                "../hello/ns_pkg/hello/test_hello.py::test_hello PASSED *",
```

Comment:
this indicates a breaking change

Reply:
Node ids are documented to be relative to the rootdir, so this looks like a bugfix to me.

Reply:
some bugfixes are breaking changes, based on the required output change here, this is a breaking change

i wll have to crosscheck as im realatively sure that this change would break reporting continuitiy in at least one work project

Reply:
ping @RonnyPfannschmidt